netapp-ontap-sql/scripts/jumpvmScript.ps1

```diff
@@ -0,0 +1 @@
+Get-ScheduledTask -TaskName ServerManager | Disable-ScheduledTask -Verbose

```

Comment:
is this used?  The CSE that disables SvrMgr does not reference this file...

Reply:
removed.. wasnt being used